### ISSUE TITLE
Subsequent database initialisations make play fail

### ISSUE THREAD

**Issue by raneq** [coopdevs/odoo-role#40]
**Description**
Task `Init Odoo database` will exit with non-zero code if executed again after a first successful run (my guess)

**Expected behaviour**
We don't try to init the db if it is already initialised.

**Related code**

https://github.com/coopdevs/odoo-role/blob/07e42559a385f4fdb0801e1818491632cc68cfee/tasks/main.yml#L112-L116

**Error message**
*Summarised*
```
TASK [coopdevs.odoo-role : Init Odoo database] 
fatal: [odoo.local]: FAILED! => {
    "changed": true,
    "cmd": [
        "/opt/.odoo_venv/bin/python", "/opt/odoo/odoo-bin",
        "-c", "/etc/odoo/odoo.conf", "-d", "odoo", "--init",
        "crm,project,stock, ··· ,phone_validation", "--stop-after-init"
    ],
    "delta": "0:02:17.448641",
    "end": "2019-02-27 14:44:43.793446",
    "msg": "non-zero return code",
    ···
}
```

*Complete*

```
TASK [coopdevs.odoo-role : Init Odoo database] **********************************************************************************************************************************
 [WARNING]: Using world-readable permissions for temporary files Ansible needs to create when
becoming an unprivileged user. This may be insecure. For information on securing this, see
https://docs.ansible.com/ansible/become.html#becoming-an-unprivileged-user

fatal: [odoo.local]: FAILED! => {"changed": true, "cmd": ["/opt/.odoo_venv/bin/python",
"/opt/odoo/odoo-bin", "-c", "/etc/odoo/odoo.conf", "-d", "odoo", "--init",
"crm,project,stock,sale_management,stock_account,base_address_city,crm_project,hr_timesheet,mail,
hr_holidays,account,account_invoicing,account_asset,purchase,hr,hr_expense,
account_analytic_default,account_bank_statement_import,account_cancel,analytic,auth_crypt,
auth_signup,barcodes,base,base_automation,base_iban,base_import,base_setup,base_vat,
base_vat_autocomplete,board,bus,calendar_sms,contacts,decimal_precision,document,fetchmail,
google_account,google_calendar,google_drive,google_spreadsheet,hr_contract,iap,l10n_es,payment,
payment_transfer,procurement_jit,product,product_margin,project_timesheet_holidays,resource,sale,
sale_crm,sale_expense,sale_margin,sale_stock,sale_timesheet,sales_team,sms,utm,web,
web_diagram,web_editor,web_kanban_gauge,web_planner,web_settings_dashboard,web_tour,
calendar,portal,http_routin,crm_phone_validation,phone_validation", "--stop-after-init"],
"delta": "0:02:17.448641", "end": "2019-02-27 14:44:43.793446", "msg": "non-zero return code",
"rc": 255, "start": "2019-02-27 14:42:26.344805", "stderr": "", "stderr_lines": [], "stdout": "", stdout_lines": []}
```

**Comment by enricostano** [comment]
@raneq would you mind to run the playbook with `-vvvv` verbosity and paste the relevant output to a [gist](https://gist.github.com)? Thanks!

**Comment by enricostano** [comment]
@raneq this should be solved by #53 right?
